src/pybind/mgr/rgw/cli.py

```diff
@@ -0,0 +1,222 @@
+#!@Python3_EXECUTABLE@
+# -*- mode:python -*-
+# vim: ts=4 sw=4 smarttab expandtab
+#
+# Processed in Makefile to add python #! line and version variable
+#
+#
+
+import subprocess
+import random
+import string
+import json
+import argparse
+import sys
+import socket
+import base64
+import logging
+
+from urllib.parse import urlparse
+
+from .rgwam import RGWAM, EnvArgs
+from .types import RGWAMEnvMgr, RGWAMException
+
+class RGWAMCLIMgr(RGWAMEnvMgr):
+    def __init__(self):
+        pass
+
+    def apply_rgw(self, svc_id, realm_name, zone_name, port = None):
+        return None
+
+    def list_daemons(self, service_name, daemon_type = None, daemon_id = None, hostname = None, refresh = True):
+        return []
```

Comment:
why can't this 'cli' stuff apply changes or list daemons, compared to `RGWAMOrchMgr` which can? what's the consequence of disabling these here?

Reply:
The cli stuff is non containerized env, not tied to the orchestrator. Getting the list of daemons is needed so that we could reconcile orchestrator config.